subs_app_versions.py

```diff
@@ -0,0 +1,65 @@
+"""
+This is a script intended to be run in a Django shell. It substitutes
+invalid build versions in Application.built_with.version.
+
+Context: https://dimagi.atlassian.net/browse/SAAS-18157
+"""
+from couchdbkit.exceptions import BadValueError
+from corehq.apps.app_manager.models import Application
+from corehq.apps.builds.models import SemanticVersionProperty
+from corehq.apps.es import filters
+from corehq.apps.es.apps import AppES
+
+VERSION_SUBSTITUTIONS = {
+    # From https://github.com/dimagi/commcare-hq/pull/20845
+    '1.2.dev': '1.2.999',
+    '1.2.0alpha': '1.2.998',
+    '1.3.0alpha': '1.3.999',
+    '2.25.w': '2.25.999',
+}
+
+
+def subs_app_versions():
+    modified = []
+    for hit in (
+        AppES()
+        .doc_type('Application')
+        .is_build(build=False)
```

Comment:
( I have very little knowledge of our `Application` class. ) `build=False` means we will return any `Application` that has an empty `copy_of` field. So it means the first version of the app. Is my understanding correct?